apps/routecraft.dev/src/app/docs/reference/operations/dedupe/page.md

```
@@ -34,16 +31,30 @@ craft()
+
+// Bound memory on a long-running route with a TTL
+craft()
+  .id('idempotent-consumer')
+  .from(queue())
+  .dedupe({ key: e => (e.body as { eventId: string }).eventId, ttl: 3_600_000 }) // remember keys for 1h
+  .process(handleEvent)
+  .to(destination)
 ```
 
 **Options:**
 - `key` (optional) - Function to derive the deduplication key from the exchange. If omitted, a key is derived by hashing the exchange body. See [default key derivation](#default-key-derivation).
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)